packages/core/src/util/index.ts

```diff
@@ -127,11 +122,11 @@ export { composePaths, composeWithUi, Paths, toDataPath };
 
 // Runtime --
 const Runtime = {
-  isEnabled(props: StatePropsOfRenderer, state: JsonFormsState): boolean {
-    return isEnabled(props, state);
+  isEnabled(uischema: UISchemaElement, state: JsonFormsState): boolean {
```

Comment:
this is an api change, should we not keep backward compatibility?

Reply:
As discussed, I wouldn't, because I think the signature as it is right now does not make much sense. I'd rather just document it in our release section. I've also updated the signature as passing the whole state as the second parameter is weird as well.